### PR TITLE
Submitted a new extension to the gallery

### PR DESCRIPTION
**Pull request by ernstc**
The extension submitted is "SQL Data Inspector" (sql-data-inspector).

This PR fixes #

### CODE REVIEW

**Review by Charles-Gagnon — CHANGES_REQUESTED** [review]
Could you please : 

1. Add this to the [insiders gallery](https://github.com/microsoft/azuredatastudio/blob/release/extensions/extensionsGallery-insider.json) as well
2. Add a `Microsoft.AzDataEngine` property next to the `Microsoft.VisualStudio.Code.Engine` property (it can still be * since it doesn't look like you have any dependency on a specific version of ADS, but you should still have one regardless)

**Review by Charles-Gagnon — APPROVED** [review]
Played around with it a bit and didn't see any major issues, @alanrenmsft this look good to you?

**Review by Charles-Gagnon — APPROVED** [review]

**Review by alanrenmsft — APPROVED** [review]

File: extensionsGallery-insider.json [hunk 1]
```diff
@@ -3940,6 +3940,67 @@
 					],
 					"statistics": [],
 					"flags": ""
+				},
+				{
+					"extensionId": "80",
+					"extensionName": "sql-data-inspector",
+					"displayName": "SQL Data Inspector",
+					"shortDescription": "This extension simplify the inspection of data stored in tables and columns to just few clicks.",
```
Comment (alanrenmsft) [inline extensionsGallery-insider.json:3948]: nit: is it really necessary to mention `stored in tables and columns`? especially `the columns` doesn't sound correct to me.

Comment (alanrenmsft) [reply extensionsGallery-insider.json:3948]: we can merge the PR if you want to leave the description as is.

**Review by kburtram — APPROVED** [review]

File: extensionsGallery-insider.json (hunk 1 above)
Comment (ernstc) [reply extensionsGallery-insider.json]: Yes, thanks. I have changed the description. Now it should sound better. In the meantime I have fixed some bugs and increased the version.

**Review by Charles-Gagnon — APPROVED** [review]

**Comment by Charles-Gagnon** [comment]
@ernstc The gallery has been updated, thank you for your contribution!

**Pull request merged**